securedrop_client/conversation/transcript/items/file.py

```diff
@@ -0,0 +1,13 @@
+from securedrop_client import db as database
+
+from .item import Item
+
+
+class File(Item):
+    type = "file"
+
+    def __init__(self, record: database.File):
+        super().__init__()
+
+        self.filename = record.filename
```

Comment:
@rocodes and @gonzalo-bulnes suggest that we actually want something more like (the derivation of) the `File.location` attribute, via a slight refactoring to return a relative rather than an absolute path:

https://github.com/freedomofpress/securedrop-client/blob/f55606fc16d52943dbafc3905a87f53ca5e34889/securedrop_client/db.py#L390-L404

Reply:
@cfm I realized that while the file structure on disc places the files in directories named after the source designation (and a `*-doc` suffix), that structure is not reproduced when the file is exported! (And maybe that's what @rocodes had in mind all along when preferring `File: memo.txt` :confused:)

Instead, the file is placed at the root of the export directory (which is created and timestamped for each export, so it isn't possible to overwrite a previous export). As a result, the correct reference would be: `File: memo.txt` like it is now.

However, that structure is not viable as-is for exporting more than one file. It breaks as soon as two files share the same name, like the many `memo.txt` that we use in dev environment.

Where does that leave us?

- until we export the files alongside the conversation transcript, there is no way to reference them from the transcript (the timestamps are not predictable)
- once we export more that one file at once, we need to define a naming scheme that prevents file name collisions, and use those names as references in the transcript
- for now, anything is probably OK, I don't see how one makes more sense than the other, the transcript simply doesn't reference any file in disk :slightly_frowning_face: